src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+
+    String deadline;
+
+    public Deadline(String description, String deadline) {
+        super(description);
+        this.deadline = deadline;
+
+    }
+
+    @Override
+    public String toString() {
+        return (isDone? "[D][X] " : "[D][ ] ") + description + (". Deadline: "  + this.deadline);
```

Comment:
minor nit about spacing which was mentioned in the coding standard.  this can be also be found in other similar methods.
```suggestion
    public String toString() {
        return (isDone ? "[D][X] " : "[D][ ] ") + description + (". Deadline: "  + this.deadline);
```